server/src/infra/repositories/asset.repository.ts

```diff
@@ -429,6 +429,17 @@ export class AssetRepository implements IAssetRepository {
     return result;
   }
 
+  getRandom(ownerId: string, count: number): Promise<AssetEntity[]> {
+    // can't use queryBuilder because of custom OFFSET clause
+    return this.repository.query(
+      `SELECT *
+                                  FROM assets
+                                  WHERE "ownerId" = $1
+                                  OFFSET FLOOR(RANDOM() * (SELECT COUNT(*) FROM ASSETS)) LIMIT $2`,
```

Comment:
Looks like this could return less than the limit in a few situations if the offset was close to the number of assets. Probably not a big deal for most situations though.

Reply:
You're right, I didn't think about boundary values. I think I fixed it in a more or less proper way.